src/app/components/address-identicon/address-identicon.component.css

```diff
@@ -0,0 +1,5 @@
+.container__identicon {
+    border-radius: 50%;
+    height: 100%;
+    cursor: default;
```

Comment:
Why did you introduce that? Wasn't there before and I don't see why preventing the cursor to reshape due to the copy functionality and tool-tip.

Reply:
We don't show a `pointer` cursor on any of the copyable elements. We use the identicon on the contact component, that is clickable. It shows the contact actions when clicked. But they should not be shown when clicked inside the identicon, because then it copies the address.

The `default` value was here before:
https://github.com/raiden-network/webui/pull/522/files#diff-4a4927f131fed02feab8d53fc7697ae5L23

But having a visual difference on the items with copy functionality would actually be nice. I think a `cursor: copy` would be reasonable. But that should be something for another PR.